modules/core/src/main/scala/effect_zoo/contests/cdown/Kyo.scala

```diff
@@ -2,18 +2,20 @@ package effect_zoo.contests.cdown
 import effect_zoo.contests.{Cdown, Contender}
 import kyo.*
 
-
 object Kyo extends Cdown.Entry(Contender.Kyo):
-  def program: Int < Vars[Int] =
-    Vars.get[Int].flatMap: n =>
-      if n <= 0
-      then n
-      else Vars.set(n - 1).andThen(program)
+  def program: Int < Var[Int] =
+    Var
+      .get[Int]
+      .flatMap: n =>
+        if n <= 0
+        then n
+        else Var.set(n - 1).andThen(program)
 
   override def round1 =
-    Vars.run(Cdown.LIMIT):
-      for
-        a <- program
-        b <- Vars.get[Int]
-      yield (a, b)
-    .pure
+    Var
+      .run(Cdown.LIMIT):
+        for
+          a <- program
+          b <- Var.get[Int]
+        yield (a, b)
+      .eval
```

Comment:
```suggestion
Var.runTuple(Cdown.LIMIT)(program).map(_.swap)
```

The benchmark expects the resulting state to be in the second element of the tuple, which is the case for turbolift's state effect but isn't for Kyo's `Var`. It'd be nice to have a more fair return type given that this is a usability decision of the libraries.